### DISCUSSION TITLE
Option to disable backup/snapshot completely?

### DISCUSSION THREAD

**Discussion by thomaslunze** [BassT23/Proxmox#118] [General]
Hey
loving your addon but would it be possible to have an option to disable that snapshot/backup option entirely?

Thanks
Thomas

**Comment by BassT23** [comment]
This is possible in latest beta.
You can update to beta with `update beta -up` 

But please be aware, after the update you need an restart of the node.

**Comment by BassT23** [comment]
This is possible in latest beta.
You can update to beta with `update beta -up` 

But please be aware, after the update you need an restart of the node.
